components/ConnectToDiscordModal.tsx

```diff
@@ -52,11 +53,12 @@ const ConnectToDiscordModal: React.FC<ConnectToDiscordModalProps> = ({
             Connect Now
           </button>
         </NavLink>
-        <LogoutContainer>
-          <div className="text-center">
-            <a href="#">No thanks, I&apos;ll study on my own.</a>
-          </div>
-        </LogoutContainer>
+        <div
+          className="text-center"
+          onClick={/* istanbul ignore next */ () => close()}
```

Comment:
What is istanbul ignored next used for? 

If it's not needed, it should be taken out.

Reply:
If it's not there, code cov will say that line isn't covered.

Reply:
Can you please add a comment explaining that?

Reply:
Hmm so you are saying that the this line is actually covered by a test, but code cov doesn't recognize that its covered?

Reply:
line covered